src/Backend/App.vala

```diff
@@ -48,6 +49,16 @@ public class Slingshot.Backend.App : Object {
     public Synapse.Match? match { get; private set; default = null; }
     public Synapse.Match? target { get; private set; default = null; }
 
+    private Slingshot.Backend.SwitcherooControl switcheroo_control;
+
+    construct {
+        try {
```

Comment:
Looking at the `SwitcherooControl` constructor, I don't think it can throw any exceptions as you handle them with a try/catch there, so you can probably remove this one.